Devise views

lw-a opened the pull request:
Completed devise account creation/login/logout

nikkicarlos-jpn committed:
started devise

lw-a committed:
Merge branch 'devise-views' of github.com:lw-a/party-outfit-rental-app

lw-a committed:
create devise account

lw-a committed:
Merge branch 'master' of github.com:lw-a/party-outfit-rental-app

lw-a committed:
Merge branch 'master' into devise-views

lw-a committed:
complete user creation/login/logout

mthisgaard commented:
Looks good to me :-)
Maybe add "unique: true" to the email validations? 
Alos we are still missing the whitelisting in the Applications controller. Will you add that too?

White-list approach
Protect every route by default:

# app/controllers/application_controller.rb
class ApplicationController < ActionController::Base
  before_action :authenticate_user!
end

lw-a committed:
add whitelist

lw-a commented:
> Looks good to me :-) Maybe add "unique: true" to the email validations? Alos we are still missing the whitelisting in the Applications controller. Will you add that too?
> 
> White-list approach Protect every route by default:
> 
> # app/controllers/application_controller.rb
> class ApplicationController < ActionController::Base before_action :authenticate_user! end

I've updated it with the whitelist!
By the way, email uniqueness and presence validation are automatically part of devise so I've removed them from the model.
